docs/specs/authentication-flow-api-reference.md

```diff
@@ -219,7 +232,21 @@ When you are in this step of this flow, you will see a response like the followi
             "identification": "email"
           },
           {
-            "identification": "phone"
+            "identification": "phone",
+            "risk_assessment": {
+              "required": true,
+              "provider": {
```

Comment:
Really a question: do we really need `type` and `alias`? In our use case probably impossible to support say two set of credentials of same type?

So we could keep it simple to just have one identifier for provider?

Reply:
So something like....

```
"captcha": {
    "required": true,
    "provider": "cloudflare"
}

Reply:
I think in most case, we do not need to support two credentials of the same type. Let me remove `alias` for now. We can add it in the future if we need it.